pages/common/git-blame.md

```diff
@@ -10,3 +10,7 @@
 - Print file with author email and commit hash on each line:
 
 `git blame -e {{file}}`
+
+- Run git blame on file as it was before a specified commit:
```

Comment:
While it may be better to spell out the whole description, the main intention of this addition was to show how to blame on the file _before_ `{{commit}}`, hence the use of `~`. So I would either leave this as it is or write

> Print file with author name and commit hash on each line for the version of the file before a specific commit

which is quite lengthy.  Which version would you prefer?

Reply:
The commits are snapshots, not diffs of a file https://github.blog/2020-12-17-commits-are-snapshots-not-diffs/. So this example displays the "snapshot" of a specific commit, blaming the author of each LoC.

Reply:
While that's true, using `{{commit}}~` doesn't print the blame for the "snapshot" of `commit` but for the snapshot of the parent of `commit` due to the `~`. This is the expected behaviour, because then I see the commit `abcdef` in some line the output of `git blame` and then run `git blame abcdef` on that file, I will see `abcdef` in the same line again. Only `abcdef~` lets me see deeper into the past. So I think the explanation of this example should use the preposition "before" rather than "at".

Reply:
Oh, right! but the `~` is not required. We can add 2 example with `~` and without.

Reply:
I've pushed a version with two examples. Seems a bit redundant though, what do you think?